transit_odp/timetables/views/review.py

```diff
@@ -157,6 +161,62 @@ class LineMetadataRevisionView(OrgUserViewMixin, DetailView):
     def get_queryset(self):
         return super().get_queryset()
 
+    def get_direction_timetable(
```

Comment:
Can this function be moved to common app and imported from there as its being used in two views?